### ISSUE TITLE
Make it easier to know which result entries are common

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mvysny/aedict#278]
```
Please look at the screenshots.
Then I search for:
矛 ほこ  1: long-handled Chinese spear; lance; pike;

It gives me a result grouped together with 戈, 桙, 槍, etc. 
Which is fine.
Then Aedict tells me the meaning spear, is common.

Now, please compare with the results brought from jisho.org.
In their results, 矛 is considered common, while 戈, 桙, 槍, etc are not.

This information is actually very important to me, as I usually choose only one 
of them to add in my SRS software, and I would rather study the common version 
of the word first.

I don't think this is a limitation in Jim Breen's JMDict, because in this link:
矛(P); 鉾; 戈; 鋒; 戟; 桙; 槍 【ほこ(P); とかり(鋒)】 (n) (1) 
long-handled Chinese spear; lance; pike; (2) (ほこ only) weapon; arms; (3) 
(ほこ only) grip of a bow; (P)

It actually puts a (P) right beside the 矛 kanji.

Basically what I want is a clear way to separate the common results from their 
non-common version.

It could be an asterisk, or color, or bold, or (P).
I'm sure you'll figure something out.

```

Original issue reported on code.google.com by `diogovk@gmail.com` on 17 May 2014 at 1:23

Attachments:
- [Capture.PNG](https://storage.googleapis.com/google-code-attachments/aedict/issue-278/comment-0/Capture.PNG)
- [Screenshot_2014-05-17-10-00-15.png](https://storage.googleapis.com/google-code-attachments/aedict/issue-278/comment-0/Screenshot_2014-05-17-10-00-15.png)
- [Screenshot_2014-05-17-10-00-49.png](https://storage.googleapis.com/google-code-attachments/aedict/issue-278/comment-0/Screenshot_2014-05-17-10-00-49.png)
